sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/PythonUDF.scala

```diff
@@ -252,17 +261,24 @@ case class UnresolvedPolymorphicPythonUDTF(
  *                                 projection to evaluate these expressions and return the result to
  *                                 the UDTF. The UDTF then receives one input column for each
  *                                 expression in the list, in the order they are listed.
- * @param pickledAnalyzeResult this is the pickled 'AnalyzeResult' instance from the UDTF, which
- *                             contains all metadata returned by the Python UDTF 'analyze' method
- *                             including the result schema of the function call as well as optional
- *                             other information
+ * @param partitionByStrings String representations of the partitioning expressions before parsing.
+ * @param orderByStrings String representations of the ordering expressions before parsing.
+ * @param selectedInputStrings String representations of the selected input expressions before
```

Comment:
why do we need the string representation here?

Reply:
I reverted this part.